db/migrate/20160318170635_projects_add_extract_docker_binary.rb

```diff
@@ -0,0 +1,5 @@
+class ProjectsAddExtractDockerBinary < ActiveRecord::Migration
+  def change
+    add_column :projects, :extract_docker_packaged_artifact, :boolean, default: true, null: false
```

Comment:
when the migration is ran, what will happen to the current existing projects? Are previous values backfilled? if not won't null mean false, causing the default value of the flag to be false?